src/leap/mail/imap/account.py

```diff
@@ -49,6 +49,16 @@ def _debugProfiling(result, cmdname, start):
 # Soledad IMAP Account
 #######################################
 
+class IIMAPAccount(Interface):
+    """
+    The abstraction describing an IMAPAccount.
+    """
+
+    user_id = Attribute("""The identifier for the account (user@provider)""")
+    account = Attribute("An internal reference to a leap.mail Account, "
+                        "which is initialized with a Soledad instance""")
+
+
 class IMAPAccount(object):
```

Comment:
Should this class `@implementer(IAccount)` (or IIMAPAccount). I see it in `LocalSoledadIMAPRealm` is used as IAccount, I guess it will be nice to make the relation more explicit.